assets/src/components/ui/responsive-table/table-body.js

```diff
@@ -21,4 +22,13 @@ const TableBody = ( { children, htmlClass, ...extraProps } ) => {
 	);
 };
 
+TableBody.propTypes = {
+	children: PropTypes.element.isRequired,
```

Comment:
oooo is children considered an `element` or is it an array?

Looking on SO and various GH issues it looks like the general recommendation is:

```jsx
children: PropTypes.node,
```

which is described as ["Anything that can be rendered: numbers, strings, elements or an array (or fragment) containing these types."](https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes)

so maybe let's use that.

Make sure you search for other components that may be using this.